src/Views/Components/AdminUI/FormNavigation/index.tsx

```diff
@@ -0,0 +1,62 @@
+import {useState} from 'react';
+import {__} from '@wordpress/i18n';
+
+import styles from './style.module.scss';
+import Button from '@givewp/components/AdminUI/Button';
+import ActionMenu from '@givewp/components/AdminUI/ActionMenu';
+import LeftArrowIcon from '@givewp/components/AdminUI/Icons/LeftArrowIcon';
+import DownArrowIcon from '@givewp/components/AdminUI/Icons/DownArrowIcon';
+import {FormNavigationProps} from '@givewp/components/AdminUI/FormNavigation/types';
+
+/**
+ *
+ * @unreleased
+ */
+
+export default function FormNavigation({
+    navigationalOptions,
+    onSubmit,
+    pageDescription,
+    pageId,
+    pageTitle,
+    actionConfig,
+    isDirty,
+}: FormNavigationProps) {
+    const [toggleActions, setToggleActions] = useState(false);
+
+    const toggleMoreActions = () => {
+        setToggleActions(!toggleActions);
+    };
+
+    return (
+        <header className={styles.formPageNavigation}>
+            <div className={styles.wrapper}>
+                <button className={styles.container} onClick={() => window.history.back()}>
```

Comment:
I wonder if wouldn't be better if we had the URL for the respective List Table here instead. For instance, if you share the donation details URL with someone in your team. Once that person opens the page, its history will be empty and the button won't do anything.

Reply:
Yea if you want to pass it. Otherwise I think react-router-dom has a history.back that preserves the origin page.

Reply:
It's okay for now but let's change this in the future.